.env.local

```
@@ -0,0 +1,5 @@
+# Created by Vercel CLI
+VERCEL_OIDC_TOKEN="eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Im1yay00MzAyZWMxYjY3MGY0OGE5OGFkNjFkYWRlNGEyM2JlNyJ9.eyJpc3MiOiJodHRwczovL29pZGMudmVyY2VsLmNvbS9qMGViaW5zLXByb2plY3RzIiwic3ViIjoib3duZXI6ajBlYmlucy1wcm9qZWN0czpwcm9qZWN0OnBhcGVyY2xpcC13b3JrdHJlZS1kYXNoYm9hcmQ6ZW52aXJvbm1lbnQ6ZGV2ZWxvcG1lbnQiLCJzY29wZSI6Im93bmVyOmowZWJpbnMtcHJvamVjdHM6cHJvamVjdDpwYXBlcmNsaXAtd29ya3RyZWUtZGFzaGJvYXJkOmVudmlyb25tZW50OmRldmVsb3BtZW50IiwiYXVkIjoiaHR0cHM6Ly92ZXJjZWwuY29tL2owZWJpbnMtcHJvamVjdHMiLCJvd25lciI6ImowZWJpbnMtcHJvamVjdHMiLCJvd25lcl9pZCI6InRlYW1fcDQxZzc1TWJZVGxneXA5bE9GTUVYa1lFIiwicHJvamVjdCI6InBhcGVyY2xpcC13b3JrdHJlZS1kYXNoYm9hcmQiLCJwcm9qZWN0X2lkIjoicHJqX0hnQzdmUWV3WGtlWWtqcGthbTRsUlVrT2t4N0wiLCJlbnZpcm9ubWVudCI6ImRldmVsb3BtZW50IiwicGxhbiI6InBybyIsInVzZXJfaWQiOiI2b0J2a2VPeEg3UFJ5M2laQk1GNHVBY2kiLCJuYmYiOjE3NzUwNTgwMDEsImlhdCI6MTc3NTA1ODAwMSwiZXhwIjoxNzc1MTAxMjAxfQ.k7DAy3WscZ2HZWN2Xk-LRRBegpRyMYmWRsLN4JlT4GipBIuEHodsckvdfk3rt_MF3oTa6JVxmTV5u6szCFTu6ypi-wv6WF9hZ4ZppP0tEtWs6a047LvlMNmCyFBkmax1vK4N6lHjbqyXd0g17yg4v-u2loGqVxrIJWWXrErPfAaLkyxNX10yBQpNdmfeYxcsUsOAGkw9fskeUkTRwyEtdx2rxjWbhTUpxEPocAz22c3v6gAhVJYVwRO5-YD328m62poiNWCmDkgMGFtJYr8FhIHOl_18L2KvlP8A8gLwkvkRyep8kNixymZEKKFrhpijDHTP_HpMTneMLxDsnx_Ixw"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)